transformation-scripts/semopenalex-authors.py

```diff
@@ -323,21 +350,29 @@ def transform_gz_file(gz_file_path):
 
                 if i % 50000 == 0:
                     g.write(author_graph.serialize(format='trig'))
+                    g_rdf_star.write(rdf_start_memory)
                     author_graph = Graph(identifier=context)
+                    rdf_start_memory = ""
 
             # Write the last part
             if not i % 50000 == 0:
                 g.write(author_graph.serialize(format='trig'))
+                g_rdf_star.write(rdf_start_memory)
                 author_graph = Graph(identifier=context)
+                rdf_start_memory = ""
+            
+            g_rdf_star.write('}')  # close rdf-star file named graph
 
     f.close()
     g.close()
+    g_rdf_star.close()
 
     print(f"Worker completed .trig authors transformation with {i} lines and {file_error_count} errors")
 
     # gzip file directly with command
     # -v for live output, --fast for faster compression with about 90% size reduction, -k for keeping the original .trig file
     os.system(f'gzip --fast {trig_output_dir_path}/{gz_file_name}.trig')
+    os.system(f'gzip --fast {trig_output_dir_path}/{gz_file_name}-rdf-star-triples.trig')
```

Comment:
proper rdf star naming is `.trigs`
```suggestion
    os.system(f'gzip --fast {trig_output_dir_path}/{gz_file_name}.trigs')
```